plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/PinotQueryBuilder.java

```diff
@@ -25,6 +25,8 @@
 import io.trino.spi.type.RealType;
```

Comment:
A truer commit message would be `Add support for pushdown of varbinary filters to Pinot`.